community/blog/2023-05 New Azure Logic Apps and DevOps functionality in milestone v1.0 Arcus Scripting release/README.md

```diff
@@ -0,0 +1,71 @@
+
+🔗 For more information on this particular command and other Logic Apps-related commands in this module, [see our feature documentation](https://scripting.arcus-azure.net/Features/powershell/azure-logic-apps).
+
+## Azure DevOps
+There is not even an Azure DevOps package that lets you interact with DevOps-related custom functionality. Interacting with DevOps functionality requires constant research for the correct syntax and possibilities. Luckily, with Arcus Scripting we have bundled commonly used functionality in the `Arcus.Scripting.DevOps` module which the v1.0 release enhances even further. 
+
+### Set a variable as secret
+Setting a secret in Azure DevOps dynamically at runtime requires you to write a specially designed host message. Because of this fragile way of working, we already had a command that was able to do this for you. This release now adds the option to dynamically write a variable as a secret. This means that it is not publicly logged in the build output so is handled with more care.
+
+```powershell
+PS> Set-AzDevOpsVariable "my-variable" "my-variable-value" -AsSecret
+##vso[task.setvariable variable=my-variable;issecret=true]***
+```
+
+🔗 For more information on this particular command and other DevOps-related commands in this module, [see our feature documentation](https://scripting.arcus-azure.net/Features/powershell/azure-devops).
+
+### Set a custom retention period
+We were already able to programmatically retain an Azure DevOps build indefinitely. In some scenarios, though, indefinite is too long. Especially if you have to delete them manually. The new v1.0 release will let you pass in the number of days such a build should be saved, making the command more flexible for numerous scenarios.
+
+```powershell
+PS> Save-AzDevOpsBuild `
+  -ProjectId $(System.TeamProjectId) `
+  -BuildId $(Build.BuildId) `
+  -DaysToKeep 10
+# Saved Azure DevOps build for 10 days with build ID $BuildId in project $ProjectId
+```
+
+💡 The variables `$(System.TeamProjectId)` and `$(Build.BuildId)` are predefined Azure DevOps variables. Information on them can be found [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml).
+
+## Conclusion
+We have released Arcus Scripting v1.0! 🎉
+
+This new major release has besides some great new and enhanced functionality a big role to play. From now on we are confident that the Arcus Scripting library is ready for the public. It has been used across several projects, has received many enhancement and feature requests, which makes it mature enough for us to decide to take this milestone of a step.
```

Comment:
```suggestion
Besides some great new and enhanced functionality this new major release has a big role to play. We are confident that the Arcus Scripting library is ready for the public. It has been used across several projects, has received many enhancement and feature requests, which makes it mature enough for us to decide to take this milestone of a step.
```

I'm not entirely sure about `We are confident that the Arcus Scripting library is ready for the public`, as that sort if implies that is wasn't in the previous releases (which I think it was 😉). Maybe something like `We are confident that the Arcus Scripting library is mature enough to move to an official v1.0 release`?

Reply:
Yes, that would be better formulated. Thx!

Reply:
Add your suggestion here: https://github.com/arcus-azure/arcus/pull/327/commits/6d4b073e7b52c2f83571417752b506d3daf4adba